File: 107598019/HW2/Software Architecture_XML/Software Architecture/adapter/presenter/AddCoursePresenter.cs
using Software_Architecture.usecase.addCourse;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Software_Architecture.adapter.presenter
{
    public class AddCoursePresenter : AddCourseOutput
    {
        public AddCoursePresenter() { }

        private string title;
        private ViewModel vm = new ViewModel();

        public string getTitle()
        {
            return title;
        }

        public void setTitle(string title)
        {
            this.title = title;
            vm.setTitle(title);
        }

        public ViewModel getViewModel()
        {
            return vm;
        }
    }
}

File: 107598034/HW2/CourseSystem/src/Adapter/Presenter/AddCoursePresenter.java
package Adapter.Presenter;

import CourseUseCase.AddCourseUseCase.AddCourseOutput;

public class AddCoursePresenter implements AddCourseOutput {
	
	private String courseId = "";
	@Override
	public String getCourseId() {
		return courseId;
	}
	
	@Override
	public void setCourseId(String courseId) {
		this.courseId = courseId;
	}
	
	
}
